Makefile

```diff
@@ -42,8 +42,10 @@ all: crystal
 crystal: pokecrystal.gbc
 crystal11: pokecrystal11.gbc
 
+# TODO: clean bitmask.asm and frames.asm for all mons except unown
 clean:
 	rm -f $(roms) $(crystal_obj) $(crystal11_obj) $(roms:.gbc=.map) $(roms:.gbc=.sym)
+	find . \( -iname '*.1bpp' -o -iname '*.2bpp' -o -iname '*.lz' -o -iname '*.gbcpal' -o -iname '*.dimensions' -o -iname '*.animated.tilemap' \) -exec rm {} +
```

Comment:
Please add "make mostlyclean" to not remove the graphics; those take so long to rebuild.

Reply:
I've always used the following commands for this:
```
	find gfx \( -name "*.[12]bpp" -o -name "*.lz" \) -delete
	find gfx/pokemon -mindepth 1 \( -name "bitmask.asm" -o -name "frames.asm" -o -name "front.animated.tilemap" -o -name "front.dimensions" -o -name "normal.gbcpal" \) -delete
	find gfx/trainers gfx/sprites -name "*.gbcpal" -delete
```